examples/lighting-app/mbed/README.md

```diff
@@ -0,0 +1,235 @@
+# CHIP ARM-MbedOS  Lighting Example Application
```

Comment:
Missing 'MbedOS logo' - it would be a good idea to use the same style for all README files across one platform.

Reply:
done